packages/styles/scss/components/slider/_slider.scss

```
@@ -146,7 +146,7 @@
       block-size: convert.to-rem(2px);
       content: '';
       inline-size: convert.to-rem(6px);
-      inset-block-start: calc(50% - #{convert.to-rem(2px) / 2});
+      inset-block-start: calc(50% - #{calc(convert.to-rem(2px) / 2)});
```

I don't think this will properly interpolate the function call. It might be easier to just multiply by half:

```suggestion
      inset-block-start: calc(50% - #{convert.to-rem(2px) * .5});
```

---

great solution. thanks